docs/SINK-ROUTES.md

```diff
@@ -0,0 +1,12 @@
+# Entur Routes
+
+Stops- and Timetable data
+
+## Data
+
+Provides timetables and stop places. The stops referred to are defined in the national stop registry (NSR), which are imported in the `Entur Stops` sink. The stop identifiers are unique across the datasets, regardless of data source 
+
+
+## Source
+
+Entur is the national registry for all public transport in Norway. The registry contains data about daily departures and routes. This including name, type, latitude, longitude, valid to/from date
```

Comment:
```suggestion
Entur is the national registry for all public transport in Norway. The registry contains data about daily departures and routes. This sink includes name, type, latitude, longitude and valid to/from date.
```

Reply:
Rewritten to: **This includes name, type, latitude, longitude and valid to/from date.** Since it is more about what kind of data the source has, and not what the sink actually imports (that's for the data paragraph)

Reply:
Actually rewritten to: **This includes name, type, latitude, longitude, valid to/from date and stop places.** Since that is more correct